### DISCUSSION TITLE
Waiting for student benefits

### DISCUSSION THREAD

**Discussion by Eskimo93** [community/community#131937] [GitHub Education]
### Hi everyone,

On July 4th, my request for github education benefits was approved but benefits are not available yet. What should I do?

**Comment by MubarakNative** [comment]
It usually takes up to **72 hours** for benefits to activate after approval: But you have crosses the time.
You can Explore **GitHub Campus**: While waiting, you can still benefit from GitHub Campus by connecting with other students, attending events, and exploring student-created repositories.
Contact GitHub Support:
Reach out to GitHub support directly for assistance. They can investigate the issue and provide specific guidance.
Links:
GitHub Education Support:
[github.com](https://github.com/orgs/community/discussions/49366)
GitHub Campus:
[github.blog](https://github.blog/2016-06-25-github-campus-experts-technology-leadership-at-your-school/)

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.
